plat/rockchip/rk3368/drivers/pmu/pmu.c

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright (C) 2015, Fuzhou Rockchip Electronics Co., Ltd
+ * Author: Tony.Xie
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
```

Comment:
We cannot include any GPL licensed code in the project. Please confirm you own the copyright and then modify all header files like this to use the 3-clause BSD boilerplate license text that the rest of the codebase uses.